demo/full/bxf-parser2/src/parser/schedule/schedule_elements/non_primary/index.js

```diff
@@ -0,0 +1,32 @@
+import parseOverlay from "./overlay";
+import reduceChildren from "../../../reduce_children";
+
+export function parseNonPrimaryEvent(parentNode) {
+  return reduceChildren(parentNode, (res, name, node) => {
+    switch (name.toLowerCase()) {
+      case "ns1:nonprimaryevent":
+        res = reduceChildren(node, (res, name, node) => {
+          switch (name.toLowerCase()) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: "switch" statements should have at least 3 "case" clauses ([javascript:S1301](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3AS1301))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=366&issues=AWbt7Z3uW3Y69bz5aGXF&open=AWbt7Z3uW3Y69bz5aGXF)

[ISSUE_KEY]: AWbt7Z3uW3Y69bz5aGXF